.github/workflows/tests.yml

```
@@ -112,6 +112,32 @@ jobs:
         run: |
           asv check -v -E existing
 
+  test_skimage_linux_free_threaded:
+    name: linux-cp313t-default
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout scikit-image
+        uses: actions/checkout@v4
+
+      # TODO: replace with setup-python when there is support
+      - uses: deadsnakes/action@e704e592227156e253e24c7f7ccf2bfe62ef58d4 # v3.1.0
```

```suggestion
      - uses: deadsnakes/action@6c8b9b82fe0b4344f4b98f2775fcc395df45e494 # v3.1.0
```
I think this is the "wrong" hash from `main` instead of the release. I presume we also have to whitelist this action in our project config. @stefanv, I assume we just trust this action implicitly because other projects already use it? It's not clear to me who's behind this...

---

It's by the flake8 / pre-commit author, so I think it's fine.